src/app/routes/regex/index.js

```diff
@@ -2,6 +2,14 @@ import services from '../../lib/config/services/loadableConfig';
 
 const serviceRegex = Object.keys(services).join('|');
 const idRegex = 'c[a-zA-Z0-9]{10}o';
+const serviceIdRegex = [
+  'bbc_amharic_radio',
+  'bbc_oromo_radio',
+  'bbc_indonesian_radio',
+  'bbc_korean_radio',
+  'bbc_tigrinya_radio',
+].join('|');
```

Comment:
This could probably be simplified to:
```
const serviceIdRegex = `bbc_({[
  'amharic',
  'oromo',
  'indonesian',
  'korean',
  'tigrinya',
].join('|')})_radio`;
```

Reply:
That said, we could leave that change for a future refactor when we add more routes. Up to you

Reply:
Just checked this and it seems you can't nest brackets inside of param variables 👎 

❌ 
![image](https://user-images.githubusercontent.com/7791726/61783493-cc750e80-adff-11e9-9ef1-8a97633aecb3.png)

🆗 
![image](https://user-images.githubusercontent.com/7791726/61783523-db5bc100-adff-11e9-830c-a0c91b3d23ee.png)